lib/internal/bootstrap/browser.js

```diff
@@ -26,7 +26,10 @@ exposeInterface(globalThis, 'URL', URL);
 exposeInterface(globalThis, 'URLSearchParams', URLSearchParams);
 exposeGetterAndSetter(globalThis,
                       'DOMException',
-                      lazyDOMExceptionClass,
+                      () => {
+                        exposeInterface(globalThis, 'DOMException', lazyDOMExceptionClass());
```

Comment:
Why not just assign `lazyDOMExceptionClass()` to something here and return it and avoid accessing `globalThis.DOMException` again?

Reply:
Would you make add a diff suggestion for the getter please?